E/Eigen/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+
+name = "Eigen"
+version = v"3.3.7"
+
+sources = [
+    ArchiveSource("https://bitbucket.org/eigen/eigen/get/$(version).tar.bz2",
+                  "9f13cf90dedbe3e52a19f43000d71fdf72e986beb9a5436dddcd61ff9d77a3ce")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/eigen-eigen-323c052e1731
+
+mkdir build
+cd build/
+
+cmake -DCMAKE_INSTALL_PREFIX=$prefix ..
```

Comment:
Try to use the host toolchain
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} ..
```

Reply:
Tried that in the previous build. All builds failed.

Reply:
Ah my bad. It's not quite the same. Let me try.